ilamb3/tests/test_transform.py

```diff
@@ -80,4 +117,68 @@ def gen_msftmz(seed: int = 1):
 def test_transform(name, kwargs, out, value):
     transform = ALL_TRANSFORMS[name](**kwargs)
     ds = transform(DATA[name])
+    print(ds[out].mean().values)  # this value should be the correct one
```

Comment:
This is helpful to figure out what the value is to check against, but then should be removed in the commit.